api/graphql/schema.graphql

```diff
@@ -1795,7 +1794,7 @@ type Mutation {
     updateUserRolesInput: UpdateUserRolesInput! @spread
   ): UserAuthInfo
     @update(model: "User")
```

Comment:
If we change the UpdateUserRolesInput, we'll need to remove the `@update` directive which lets Lighthouse magically handle the update, and do it in a mutation class.